lib/content-services/src/lib/search/components/search-date-range-advanced-tabbed/search-date-range-advanced/search-date-range-advanced.component.html

```diff
@@ -0,0 +1,43 @@
+<ng-container [formGroup]="form">
+  <mat-radio-group formControlName="dateRangeType">
+    <span class="adf-search-data-range-horizontal-container">
```

Comment:
I think this class has typo. You have here data but in scss you have date

Reply:
Yes, that was a typo. Nice spot. Updated the code. Thanks!